src/views/Editor/Toolbar.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react'
+import { useYMap } from '@/hooks/useYjsMap'
+import type * as Y from 'yjs'
+
+// FIXME: Needs refactoring into a more global settings file
+import { Priorities } from '@/defaults'
+import { NewsValueTimeDropDown } from './Toolbar/NewsValueTimeDropDown'
+import { NewsValueScoreDropDown } from './Toolbar/NewsValueScoreDropDown'
+
+interface ToolbarProps {
+  isSynced: boolean
+  document?: Y.Doc
+}
+
+export const Toolbar = ({ isSynced, document }: ToolbarProps): JSX.Element => {
+  const [newsvalueScore, setNewsvalueScore, initNewsvalueScore] = useYMap('core/newsvalue/score')
+  const [newsvalueDuration, setNewsvalueDuration, initNewsvalueDuration] = useYMap('core/newsvalue/duration')
+  const [newsvalueEnd, setNewsvalueEnd, initNewsvalueEnd] = useYMap('core/newsvalue/end')
+
+  useEffect(() => {
+    if (!isSynced || !document) {
+      return
+    }
+
+    const metaYMap = document.getMap('meta')
+    initNewsvalueDuration(metaYMap)
+    initNewsvalueScore(metaYMap)
+    initNewsvalueEnd(metaYMap)
+  }, [
+    isSynced,
+    document,
+    initNewsvalueDuration,
+    initNewsvalueScore,
+    initNewsvalueEnd
+  ])
+
+  return (
+    <>
+      <NewsValueScoreDropDown
```

Comment:
Maybe add Tooltip for NewsValueScore and NewsvalueTime?

Reply:
👍